src/server/pkg/storage/fileset/storage.go

```diff
@@ -272,7 +283,43 @@ func removePrefix(fileSet string) string {
 	return fileSet[len(prefix):]
 }
 
+func removePrefixes(xs []string) (ys []string) {
```

Comment:
Would you mind switching this to not using a named return. Although this is a simple case, unnecessarily using named returns can lead to subtle bugs. I think in general we should only be operating on named return variables when handling errors.